src/core/sal/qemu/QEMUController.cc

```diff
@@ -18,7 +18,7 @@ QEMUController::QEMUController()
 QEMUController::~QEMUController()
 {
 	delete m_Regs;
-	delete m_Mem;
+	delete &getMemoryManager();
```

Comment:
At this point only the MemoryManager for MEMTYPE_RAM would be deleted, since this is the default for getMemoryManager().

Reply:
This is fine for QEMU, since there is only one kind of memor manager.